Docker

CrazyTapok opened the pull request:
1. Screenshot: 
![image](https://github.com/user-attachments/assets/343247c9-1bcd-4eff-8499-8bd8ae9b01bf)


2. What has been implemented:
  - [x] add docker file
  - [x] add docker-compose
  - [x] configure docker

CrazyTapok committed:
feat: add docker
add docker file, add docker-compose, configure docker-compose

CrazyTapok committed:
fix: change docker-compose
Database added in a separate container.

svirzhevskiy commented on src/docker-compose.yml:
```diff
@@ -0,0 +1,26 @@
+      - "8081:8081"
+    environment:
+      - ASPNETCORE_ENVIRONMENT=Development
+      - ConnectionStrings__DefaultConnection=Server=db;Database=TaskManager_db;Integrated Security=True;
```
ты уверен, что твое приложение коннектится именно к базе в докере?

CrazyTapok committed:
fix: change docker-compose
change ConnectionStrings, add depends in docker-compose taskmanager container, add names for containers, add healthcheck for db.
